### DISCUSSION TITLE
The new dx build/serve interface in 0.6.x is great

### DISCUSSION THREAD

**Discussion by ThomasCartier** [DioxusLabs/dioxus#2770] [General]
Just a quick note for the devs behind the new interface when building the app.
It's really handy to be able to switch between console and build, in case of an error, to gather as much info as possible in a non timed fashion.
Really clear as well to know early on where the server listens now that port and addr can be passed as arguments.

The web interface during the build brings some welcomed UX.

Overall, it's great. And one can feel it when going back to v0.5.6 for prod use.

Thanks

**Comment by jkelleyrtp** [comment]
Thanks! We're still tuning it / making it faster / adding new features.

Do you have any ideas for other features we could add to it?

**Reply by ThomasCartier** [reply]
Hi!
As for the interface, it's all good.

The only missing piece is a way for `dx` to gather the translations in each sub crates for the project and merging them automatically in the app.

It means that each crate using dioxus should have a translation folder created during the `dx init/new` or manually by the user, and the user should put his translations there.

That way we can split one big app in multiple ones.

That's a missing piece. Now it's way too manual.

**Reply by DogeDark** [reply]
We could probably integrate translation propagation through Manganis. It solves the same issue except for stylesheets and other assets.

**Reply by ThomasCartier** [reply]
That would be really nice indeed!
